### ISSUE TITLE
Clarify working group membership

### ISSUE THREAD

**Issue by ericsampson** [dotnet-foundation/ecosystem-growth#11]
- is the working group membership invite-only?
- is the WG closed to new members for the time being?
- is there a mechanism to volunteer for the group? Or is it by invite of the WG lead(s). Either way, would be good to have documented.

Cheers!

**Comment by oskardudycz** [comment]
I agree - I heard a few times that "MSFT is working with DNF on something" but I haven't seen any information neither on the slack nor other channel about how to join the group, how to provide the feedback or how to see details of what's exactly has been done.

I understand that it's more effective to work in smaller working groups, but it would be good to clarify what @ericsampson mentioned. There are many ways how to get other people involved. Such a meeting can be streamed live or recorded and watched on-demand, there might be meetings notes written. Issues on Github where we have a discussion.

As a DNF member from over one year and a co-maintainer of DNF project, I don't see any open discussion, any transparency and proactive attitude from the DNF of working with the members and in the general whole community.

Especially such important topics like collaboration, sustainability etc. should be (in my opinion) bottom-up initiatives and start from the maintainers, devs to get their feedback. I might miss (however unlikely) but I haven't seen any survey or discussion what'd help maintainers to make OSS their work sustainable and effective. We get the document (https://github.com/microsoft/dotnet/blob/master/docs/ecosystem-issues.md) that we cannot comment and closed working group with no notes.

I apologize if someone finds that feedback negative but they're many ways and tools how to make it more transparent, effective and leverage them to building trust to DNF and MSFT. We just need to use them.

**Comment by sbwalker** [comment]
I am happy to provide some transparency on the genesis of this working group...

The[ initial document](https://github.com/microsoft/dotnet/blob/214c8c343587461f161198cdf5e9084abddde179/docs/ecosystem-issues.md) was written by Immo Landwerth and Eric Erhardt as part of a planning exercise for .NET 6. Immo has taken on the responsibility as a Microsoft PM to drive the problem definition and provide a starting point so that we can work together as a community to come up with solutions. 

To facilitate the collaboration, Immo reached out to the Executive Director of the .NET Foundation, Claire Novotny, to determine if there was an opportunity to work together on this initiative. Claire directed Immo to the Project Committee, as it is already well established as a functional .NET Foundation committee and is already well versed on many of the topics outlined in Immo's document. 

One of the conditions of participating in this working group was a time commitment. Immo suggested that the working group should meet weekly for 2 hours for the next 3-4 months. The Project Committee is comprised entirely of volunteers who already contribute a significant amount of time to the .NET Foundation... so we needed to ensure that we had enough members who were willing/able to make this additional time commitment. We had 4-5 members who confirmed that they could so we had a kickoff meeting in November.

The kickoff meeting was focused on reviewing the document, soliciting initial feedback, delegating various problem areas to group members, and discussing the mechanics of how the group should operate. In order for the group to be effective it was decided that we should keep its membership to a manageable size but also ensure enough voices are represented. We had 8 members to start with and we felt the total should not exceed 15 members. We also agreed that the activities of the group need to be as transparent as reasonably possible ( keeping in mind that there are always some topics which are more sensitive in nature that may need to be handled privately ).  

In order to broaden the membership of the group we felt it made sense to include more representatives from the .NET Foundation. Specifically we focused on maintainers of .NET Foundation member projects, as we felt these individuals would have the most practical experience and perspective on the challenges we are facing. We reached out to a number of maintainers. Some accepted and others declined to participate because they felt they did not have the time to devote to this effort. The current list of group members is listed on the home page of this repo.

The working group is scheduled to meet every Thursday from 4-6 PM ET for the next 3-4 months. The next meeting will be after the holidays... on Jan 7.

**Comment by sbwalker** [comment]
@oskardudycz the working group openly admits it does not have the answers to all of these challenges. We will actively engage with the .NET community to solicit ideas, proposals, feedback, etc... I would expect that much of that engagement will occur here in this repository. There is not much to see yet because we are only just getting started with this initiative. We fully expect that given the current climate in the .NET ecosystem there will be a lot of public venting of frustrations, however the main role of the working group is to encourage the community to channel their passion and energy into practical solutions which can make a positive difference for the future.

**Comment by oskardudycz** [comment]
@sbwalker thank you for the answer, I appreciate that. What's the best way to contribute or send feedback?

**Comment by sbwalker** [comment]
As I mentioned, the working group is just getting started so we are still determining an effective process. That being said, if you have ideas or feedback I would encourage you to post them publicly in this repo. Since Discussions are not enabled yet, please create a new Issue. This way the working group as well as the entire community can participate. I created the initial [Issue](https://github.com/dotnet-foundation/ecosystem-growth/issues/7) to share my own thoughts and allow others to comment.
